neuralmonkey/trainers/regularizers.py

```diff
@@ -104,36 +118,35 @@ def __init__(self,
                 on the previous task.
         """
         check_argument_types()
-
         Regularizer.__init__(self, name, weight)
 
-        if gradients_file is None:
-            raise ValueError("Missing gradients_file")
-        if variables_file is None:
-            raise ValueError("Missing variables_file")
-
-        log("Loading initial variables for EWC from {}".format(variables_file))
+        log("Loading initial variables for EWC from "
+            "{}.".format(variables_file))
```

Comment:
konkatenaci stringů nedělej a druhej řádek (pokud se to fakt nevejde na jednu) začni `.format`.